apps/api/src/domain/models/transcribed-audio/entities/transcribed-audio.entity.ts

```diff
@@ -33,4 +38,29 @@ export class TranscribedAudio extends Resource {
 
         this.transcript = new Transcript(transcriptDto);
     }
+
+    validateTimeRangeContext({ timeRange }: TimeRangeContext): Valid | InternalError {
+        const { inPoint, outPoint } = timeRange;
+
+        /**
+         * Note that the `startMilliseconds` doesn't have to be 0, so unlike the
+         * geometric models, we do need to confirm here that `inPoint` isn't too low.
+         */
+        if (
+            [inPoint, outPoint].some(
+                (timePoint) => !isNumberWithinRange(timePoint, this.getTimeBounds())
+            )
+        )
+            return new InconsistentTimeRangeError(timeRange, this);
```

Comment:
Readability is a bit off here just because of the nested brackets.  I tried reformatting but prettier changes it back.  And it looks just as weird with the )))), heheh.  Obviously immaterial for function.

Reply:
I'll refactor by currying a baby function that gets the timeBounds from the closure in order to make the callback point free.